README.md

```diff
@@ -1,15 +1,23 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S2-CS2103-W17-1/tp/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2122S2-CS2103-W17-1/tp)
```

Comment:
URL needs to change from `https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg` to `https://codecov.io/gh/AY2122S2-CS2103-W17-1/tp/branch/master/graph/badge.svg`

Reply:
thanks for catching, resolved in upcoming commit